cronitor/__init__.py

```diff
@@ -17,6 +17,13 @@
 api_version = os.getenv('CRONITOR_API_VERSION', None)
 environment = os.getenv('CRONITOR_ENVIRONMENT', None)
 config = os.getenv('CRONITOR_CONFIG', None)
+timeout = os.getenv('CRONITOR_TIMEOUT', None)
```

Comment:
This is great!

One simplifying change I would like to consider (and am open to feedback on), is to simply set
`timeout` here, and then read that internally in the `_put` and `as_yaml` methods instead of passing timeout as an arg to the various `*_config` methods.

I know this isn't quite what I had suggest before, but it would simplify the code and the interface a little bit.

```
import cronitor
cronitor.timeout = 30
cronitor.apply_config
```

internally the timeout attribute would just be set as, but could still be set directly on the module as done in the above example.
```
timeout = os.getenv('CRONITOR_TIMEOUT', 10)
```

Reply:
Thanks for the review! i have made the code shorter and easier to read. 
And README has also been updated. Please check!

https://github.com/cronitorio/cronitor-python/pull/26/commits/a452913aff801d1a69cc1b8970ae26c84c19606c